src/Promitor.Core/Metrics/MetricAggregationType.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using Azure.Monitor.Query.Models;
+using Microsoft.Azure.Management.Monitor.Fluent.Models;
+
+namespace Promitor.Core.Metrics;
```

Comment:
Please create a file per type, so in this case move MetricAggregationTypeConverter to a seperate file